cvprac/cvp_api.py

```diff
@@ -3665,39 +3665,130 @@ def change_control_stop(self, cc_id, notes=""):
         self.log.debug('v6 ' + str(cc_url) + ' ' + str(payload))
         return self.clnt.post(cc_url, data=payload, timeout=self.request_timeout)
 
-# Commenting this out until it's fully supported
```

Comment:
Just wanted to know if this has to be deleted. Because I thought this commented section was preserved for future use.

Reply:
yes, I've uncommented it now as it now works in the latest 2022.1.0 and CVaaS, let me up the version of this to v8

Reply:
My bad, I thought you deleted.